apps/web-giddh/src/app/vouchers/create/create.component.ts

```diff
@@ -578,7 +585,6 @@ export class VoucherCreateComponent implements OnInit, OnDestroy, AfterViewInit
                 this.searchStock();
 
                 if (this.invoiceType.isCashInvoice) {
-                    this.invoiceForm.get('account.uniqueName')?.patchValue("cash");
```

Comment:
Why removing this?

Reply:
Removing due to send deposit unique name in account unique name

Reply:
Since you are assigning account unique name before saving, this shouldn't get removed. If user won't select deposit account, this will work